cnxarchive/utils/mimetype.py

```diff
@@ -23,6 +23,7 @@
 MIMETYPES = (MODULE_MIMETYPE, COLLECTION_MIMETYPE, FOLDER_MIMETYPE,)
 PORTALTYPE_TO_MIMETYPE_MAPPING = {
     'Module': MODULE_MIMETYPE,
+    'CompositeModule': MODULE_MIMETYPE,
```

Comment:
Will it be a problem in legacy to have `portal_type = 'CompositeModule'` in the modules table?